documentation/general/dash-sonic-hld.md

```diff
@@ -265,6 +266,73 @@ It is possible that a given packet can get a hit in route table and/or mapping t
 
   ![dash-outbound-meter](./images/dash-hld-outbound-meter-pipeline.png)
 
+## 2.5 FastPath
+
+This section captures the Sonic-Dash specifics of FastPath use-case. Detailed document on FastPath is captured here (![FastPath](https://github.com/sonic-net/DASH/blob/main/documentation/load-bal-service/load-balancer-v3.md))
+The following are the salient points and requirements:
+- FastPath kicks in when appliance receives an ICMP redirect that matches an existing unified flow
```

Comment:
We need to also define the behavior of:
- What makes us consider the packet is sent to appliance?
- The behaviors when we get an ICMP packet that doesn't match any unified flow. This includes 2 parts: 1) The packet is not meant to send to us. 2) The packet is for us, but nothing matches.

Reply:
this can be part of the detailed design